scripts/gptfuzz_integration/cli.py

```
@@ -0,0 +1,209 @@
+                        help='List of harmful questions to test (default: predefined list)')
+    parser.add_argument('--output_dir', type=str, default=DEFAULT_CONFIG['output_dir'],
+                        help=f'Directory to save results (default: {DEFAULT_CONFIG["output_dir"]})')
+    
+    # Hardware settings
+    parser.add_argument('--device', type=str, default=DEFAULT_CONFIG['device'],
+                        help=f'Device to run the detector on (default: {DEFAULT_CONFIG["device"]})')
+    
+    return parser
+
+
+def main():
```

Pylint (3.3.7)

[refactor] Too many local variables (24/15)

(R0914)